taiwanese/taioan.md

```diff
@@ -0,0 +1,5 @@
+Ū chi̍t chiah gû, lí nā kā mn̄g bān hāng sèsū, i lóng kanna kā lí ìn "Bô~ Bô~".
+
+Nā beh chaiiáⁿ káitap, tio̍h ài mn̄g [Gû ko](https://www.google.com/).
+
+Chún kóng lí kámkak Gû ko bē sìntiōng tit, tiō lâi khì mn̄g [Ah á ko](https://duckduckgo.com/). Thiaⁿ kóng i [chhùipoe](http://taigi.fhl.net/dict/search.php?DETAIL=1&LIMIT=id=48709&dbname=dic&graph=2) khah ngē, bē kā lí ê pìbi̍t kóng hō͘ lâng chai.
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.